lib/godmin/engine_wrapper.rb

```diff
@@ -17,7 +17,7 @@ def namespaced?
     def namespaced_path
       @namespaced_path ||= begin
         if namespaced?
-          namespace.name.classify.split("::").map(&:underscore)
+          namespace.name.split("::").map(&:underscore)
```

Comment:
That's weird, because Rails has no issues with generating stuff in a plural engine.